web/packages/teleport/src/TopBar/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,129 @@
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import { formatDistanceToNow } from 'date-fns';
+import styled from 'styled-components';
+import { Text } from 'design';
+
+import { Notification, UserList } from 'design/Icon';
+import { useRefClickOutside } from 'shared/hooks/useRefClickOutside';
+import { useStore } from 'shared/libs/stores';
+
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownItemButton,
+  DropdownItemIcon,
+  STARTING_TRANSITION_DELAY,
+  INCREMENT_TRANSITION_DELAY,
+  DropdownItemLink,
+} from 'teleport/components/Dropdown';
+import useTeleport from 'teleport/useTeleport';
+
+import { ButtonIconContainer } from '../Shared';
+
+export function Notifications() {
+  const ctx = useTeleport();
+  useStore(ctx.storeNotifications);
+
+  const notices = ctx.storeNotifications.getNotifications();
+
+  const [open, setOpen] = useState(false);
+
+  const ref = useRefClickOutside<HTMLDivElement>({ open, setOpen });
+
+  let transitionDelay = STARTING_TRANSITION_DELAY;
+  let items = notices.map(notice => {
+    let currentTransitionDelay = transitionDelay;
+    transitionDelay += INCREMENT_TRANSITION_DELAY;
+
+    let item;
+    if (notice.kind === 'access-lists') {
+      item = (
+        <NotificationItemButton onClick={() => null}>
+          <DropdownItemIcon>
+            <UserList mt="1px" />
+          </DropdownItemIcon>
+          <Text>
+            Access list <b>{notice.resourceName}</b> needs your review within{' '}
+            {formatDistanceToNow(notice.date)}.
+          </Text>
+        </NotificationItemButton>
+      );
+    }
+
+    return (
+      <DropdownItem
+        open={open}
+        $transitionDelay={currentTransitionDelay}
+        key={notice.id}
+      >
+        <NotificationLink to={notice.route} onClick={() => setOpen(false)}>
+          {item}
+        </NotificationLink>
+      </DropdownItem>
+    );
+  });
+
+  return (
+    <NotificationButtonContainer ref={ref} data-testid="tb-note">
+      <ButtonIconContainer
+        onClick={() => setOpen(!open)}
+        data-testid="tb-note-button"
+      >
+        {items.length > 0 && <AttentionDot data-testid="tb-note-attention" />}
+        <Notification />
+      </ButtonIconContainer>
+
+      <Dropdown
+        open={open}
+        style={{ width: '300px' }}
+        data-testid="tb-note-dropdown"
+      >
+        {items.length ? (
+          items
+        ) : (
+          <Text textAlign="center" p={2}>
+            No notifications
+          </Text>
+        )}
+      </Dropdown>
+    </NotificationButtonContainer>
+  );
+}
+
+const NotificationButtonContainer = styled.div`
+  position: relative;
+`;
+
+const AttentionDot = styled.div`
+  position: absolute;
+  width: 7px;
+  height: 7px;
+  border-radius: 100px;
+  background-color: ${p => p.theme.colors.buttons.warning.default};
+  top: 10px;
+  right: 15px;
+`;
+
+export const NotificationItemButton = styled(DropdownItemButton)`
```

Comment:
IMO that 0.8 opacity doesn't look good here:
<img width="567" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/c7953338-041e-4380-aec4-a62e4791d558">

It makes the items look disabled.
I'm wondering if we need it at all, we change the background, so it is clear which item is highlighted.

Reply:
i agree, i removed the opacity